src/pages/messages/components/NviMenuContent.tsx

```diff
@@ -233,6 +234,21 @@ export const NviMenuContent = () => {
               }
             />
           </StyledNviStatusBox>
+
+          <StyledNviStatusBox sx={{ bgcolor: 'secondary.main' }}>
+            <FormControlLabel
+              data-testid={dataTestId.tasksPage.nvi.statusFilter.disputeRadio}
+              checked={nviParams.filter === 'dispute'}
+              onClick={openCandidatesView}
+              control={<StyledStatusRadio onChange={() => setNviStatusParam('dispute')} />}
+              slotProps={{ typography: { fontWeight: 700 } }}
```

Comment:
700 er vel default for "bold". Er det noen spesiell grunn til at dette er satt numerisk?

Reply:
For leeenge siden (før din tid) hadde vi hvertfall en diskusjon om dette, og ble enige om å konsekvent bruke numeriske verdier, da plana var å ha [variable fonts](https://v-fonts.com/) som støtter alle verdier (100-900) for font-weight. Vi syntes det ble tydeligere om alle hadde tall, enn at noen hadde tall mens noen var `bold` og man må vite at `bold=700`. Nå ble det ikke variable font læll da, så da har det bare blitt en blanding med begge deler som forsåvdit er fair enough 🤷